gremlin_connect/gremlin_adapter.py

```diff
@@ -64,7 +64,7 @@ def test_connection(self):
     def execute_query(self, query):
         """Execute a query on the Gremlin server."""
         _logr.debug("Executing query: {}\n".format(query))
-        response = requests.get(self.gremlin_url, params={"gremlin": query})
+        response = requests.post(self.gremlin_url, data=json.dumps({"gremlin": query}))
```

Comment:
In the requests library, you can directly pass a `dict` to the `json` parameter instead of using a `json.dumps` with the `data` parameter, for future reference. 🙂

Reply:
Actually I tried it, but few queries were failing without `json.dumps`.